src/ServerConfigParams.ts

```diff
@@ -90,9 +100,32 @@ export function verifyServerConfiguration({
   if (!app.url) throw new Error('missing param: url');
   if (!app.port) throw new Error('missing param: port');
   if (!app.workdir) throw new Error('missing param: workdir');
+
+  const {
+    disableSponsoredTx,
+    gasFeePercentage,
+    transferFeePercentage,
+    fixedUsdFee,
+  } = app;
+
+  //disableSponsoredTx should be defined
+  if (![true, false].includes(disableSponsoredTx))
+    throw new Error(ERROR_DISABLE_SPONSOR_TX_NOT_CONFIGURED);
```

Comment:
I don't this is possible, since we do include a default configuration with `default.json5`

Reply:
Agreed on put this on a separated task.